docs/configuration/monitoring.md

```diff
@@ -52,51 +149,10 @@ engine:
       enabled: true
 ```
 
-## Setting up monitoring using Crow
+Apply this configuration by issuing an helm upgrade while passing these additional values.
+To do so append `--values values-monitoring.yaml --recreate-pods` to the command used to install
+codacy provided in [this documentation](../index.md#2-installing-codacy).
 
-Crow is a visualization tool that displays information about the projects and jobs that are pending for analysis, as well as the ones running in the Codacy platform.
-
-The Crow tool is installed alongside Codacy after the helm chart is deployed to the cluster.
-
-The `crow` tool can be accessed through the `/monitoring` path of the url pointing to your Codacy installation, e.g. `http://codacy.company.org/monitoring`.
-You must set the `global.codacy.crow.url` value in your `values.yaml` file so that anchor links to your projects can be properly established inside `crow`.
-For example:
-
-```yaml
-global:
-  codacy:
-    crow:
-      url: "http://codacy.example.com/monitoring"
-```
-
-Please see the [README.md](https://github.com/codacy/chart/blob/master/README.md) for more information about these values.
-
-### Configuring your credentials
-
-You **should** provide a password for the `crow` installation either through the `values.yaml` file or through a `--set` parameter during the `helm` installation process. This parameter can be configured as follows:
-
-* Through a `--set` parameter:
-
-  ```yaml
-  helm upgrade (...) --set crow.config.passwordAuth.password=<--- crow password --->
-  ```
-
-* Through the `values.yaml` file:
-
-  ```yaml
-  crow:
-    config:
-      passwordAuth:
-        password: <--- crow password --->
-  ```
-
-**Failing to do so your `crow` will fall back to using the default credentials as described [below](##Default-credentials).**
-
-## Default credentials
-
-If you have not configured your `crow` credentials as described [above](###Configuring-your-credentials), you can login with the following default credentials:
-
-```yaml
-username: codacy
-password: C0dacy123
+```bash
+helm upgrade (...options used to install codacy...) --values values-monitoring-values.yaml --recreate-pods
```

Comment:
We have a similar `helm upgrade` command above for the Crow credentials but we aren't explaining how the user should complete the command like we do here. Should we choose to provide more detailed instructions like here, or less like above? :thinking:

Reply:
I'd rather add more info to the command above. `(...)` alone isn't often used to document command line tools, and its meaning might be obscure.